pennylane/devices/qubit/simulate.py

```diff
@@ -559,14 +569,7 @@ def simulate_tree_mcm(
             depth += 1
             # Update the active branch samples with `update_mcm_samples`
             if finite_shots:
-                if (
-                    mcms[depth]
-                    and mcms[depth].postselect is not None
-                    and postselect_mode == "fill-shots"
-                ):
-                    samples = mcms[depth].postselect * qml.math.ones_like(measurements)
-                else:
-                    samples = qml.math.atleast_1d(measurements)
+                samples = math.atleast_1d(measurements)
```

Comment:
Removes the unconditional sampling, everything else in the same file is cleaning up imports.